### PR TITLE
SP-4015 - create incident war room opsgenie

### PR DESCRIPTION
**Pull request by odedshvartz**

### CODE REVIEW

File: stackpulse/create-incident-war-room-opsgenie/playbook.yaml [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+      integrationRef: "{{ .params.slackIntegrationName }}"
+  - name: us-docker.pkg.dev/stackpulse/public/opsgenie/incident/create
+    id: opsgenie_create_incident
+    when: '{{ ieq .slack_response "Yes" }}'
+    env:
+      USE_EU_API: "{{ .params.opsgenieUseEUApi }}"
+      SERVICE_NAME: "{{ .params.pagerDutyService }}"
+      MESSAGE: "{{ .event.Name | kebabcase}}-{{.event.ID | kebabcase}}"
+      DESCRIPTION: "{{ .event.Name | kebabcase}}-{{.event.ID | kebabcase}} created by StackPulse"
+      PRIORITY: '{{ if ieq .event.Severity "normal" }}P3{{else if "high"}}P2{{ else }}P3{{ end }}'
```
Comment (hagaishapira) [inline stackpulse/create-incident-war-room-opsgenie/playbook.yaml:108]: what are we using for `critical`?

**Review by hagaishapira — APPROVED** [review]
See comments, LGTM

File: stackpulse/create-incident-war-room-opsgenie/playbook.yaml (hunk 1 above)
Comment (odedshvartz) [reply stackpulse/create-incident-war-room-opsgenie/playbook.yaml]: Fixed

File: stackpulse/create-incident-war-room-opsgenie/playbook.yaml [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+  - name: opsgenieIntegrationName
+    type: integration
+    description: Opsgenie integration.
+    options:
+      integration_type: opsgenie
+  - name: opsgenieUseEUApi
+    type: var
+    description: set to true to use Opsgenie EU region.
+    options:
+      integration_type: opsgenie
```
Comment (eldadru) [inline stackpulse/create-incident-war-room-opsgenie/playbook.yaml:36]: I think this shouldn't be here, that's a var parameter

File: stackpulse/create-incident-war-room-opsgenie/playbook.yaml [hunk 3]
```diff
@@ -0,0 +1,110 @@
+apiVersion: stackpulse.io/v1
```
Comment (eldadru) [inline stackpulse/create-incident-war-room-opsgenie/playbook.yaml:1]: In general, lets stick to the {{ $. }} syntax -> i.e. change {{ .event.Reporter }} to {{ $.event.Reporter }}

File: stackpulse/create-incident-war-room-opsgenie/playbook.yaml (hunk 2 above)
Comment (odedshvartz) [reply stackpulse/create-incident-war-room-opsgenie/playbook.yaml:36]: I copied it from slack integration parameter:
`- name: slackIntegrationName
    type: integration
    description: Slack integration.
    options:
      integration_type: slack`

File: stackpulse/create-incident-war-room-opsgenie/playbook.yaml (hunk 3 above)
Comment (odedshvartz) [reply stackpulse/create-incident-war-room-opsgenie/playbook.yaml:1]: OK, I will change both playbooks (I copied it from the original pagerduty playbook)

Comment (odedshvartz) [reply stackpulse/create-incident-war-room-opsgenie/playbook.yaml:1]: Done

File: stackpulse/create-incident-war-room-opsgenie/playbook.yaml (hunk 2 above)
Comment (odedshvartz) [reply stackpulse/create-incident-war-room-opsgenie/playbook.yaml:36]: I see it now (bad copy paste)

**Pull request merged**